things/api/things/http/requests.go

```diff
@@ -180,6 +180,13 @@ type listResourcesReq struct {
 	pageMetadata things.PageMetadata
 }
 
+type listResourcesMetaReq struct {
+	token        string
```

Comment:
Which token is this? 
User token is in req header, thing doesn't have a token, it has key (in domain language we don't call it token).